File: src/main/java/br/com/southsystem/assembleia/repositorio/AssociadoRepository.java
package br.com.southsystem.assembleia.repositorio;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import br.com.southsystem.assembleia.entidade.Associado;

@Repository
public interface AssociadoRepository extends JpaRepository<Associado, Long> {

	Optional<Associado> findByCpf(String cpf);

}

File: src/main/kotlin/br/com/southsystem/desafiobackvotos/gateway/database/AssociadoRepository.kt
package br.com.southsystem.desafiobackvotos.gateway.database

import br.com.southsystem.desafiobackvotos.domain.Associado
import org.springframework.data.repository.reactive.ReactiveCrudRepository

interface AssociadoRepository: ReactiveCrudRepository<Associado, Long> {
}